Source/DafnyLanguageServer/Workspace/VerificationProgressReporter.cs

```diff
@@ -123,7 +123,7 @@ IDiagnosticPublisher diagnosticPublisher
   /// </summary>
   /// <param name="implementations">The implementations to be verified</param>
   public void ReportImplementationsBeforeVerification(Implementation[] implementations) {
-    if (document.LoadCanceled || implementations.Length == 0) {
+    if (document.LoadCanceled) {
```

Comment:
About this: It happenes that in the past somewhere, for some time, this function was called twice, once without any implementations, and the other one with all the implementations. This started to happen after I had merged once with master, so I had this guard to prevent this. However, it would not enable to consider files without really any implementation to verify.
This "bug" does not happen anymore (probably thanks to recent's refactoring with Boogie) so I remove this check.